apps/dashboard/src/modules/seller/views/POSInfoView.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="page-container flex flex-column">
+    <div class="page-title">{{ posName }}</div>
+    <div class="flex flex-column gap-5">
+      <div class="flex flex-column md:flex-row gap-5 justify-content-between align-items-stretch w-12">
+        <POSSettingsCard :pos-id="id!" class="flex-1 h-12" />
+        <CardComponent :header="$t('c_posInfo.lastWeekSales')" class="flex-1" >
+          <div class="h-12 text-center text-5xl pb-3">{{ formattedTotalSales }}</div>
+        </CardComponent>
+      </div>
+      <ContainerCard
```

Comment:
There is no delete container from POS option :(

Reply:
Will fix in #303